src/ContaoCommunityAlliance/DcGeneral/Contao/View/Contao2BackendView/Widget/FileTreeOrder.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * PHP version 5
+ *
+ * @package    generalDriver
+ * @author     David Molineus <david.molineus@netzmacht.de>
+ * @copyright  The MetaModels team.
+ * @license    LGPL.
+ * @filesource
+ */
+
+namespace ContaoCommunityAlliance\DcGeneral\Contao\View\Contao2BackendView\Widget;
+
+/**
+ * This widget is a supporting widget to store the file tree orderings.
+ *
+ * The ContaoWidgetManager does not allow input values without a widget. This is used as helper widget instead.
+ */
+class FileTreeOrder extends AbstractWidget
+{
+    /**
+     * The template.
+     *
+     * @var string
+     */
+    protected $strTemplate = 'be_widget_rdo';
```

Comment:
Should we use an own template name here and copy the template to dcg? When there are changes in upstream, the template might not fit our implementation anymore.